components/power_agent/power_agent.py

```
@@ -0,0 +1,249 @@
+                    continue
+                
+                logger.info(f"GPU {gpu_idx} ({uuid}): {len(pids)} processes running, PIDs={pids}")
+                
+                # Map PIDs to Pod UIDs
+                pid_pod_map = self.map_pids_to_pod_uids(pids)
+                logger.info(f"GPU {gpu_idx} ({uuid}): PID-to-Pod mapping: {pid_pod_map}")
+                
+                # Check if any process belongs to a pod with power limit
+                target_limit = None
+                target_pod_uid = None
+                for pid, pod_uid in pid_pod_map.items():
```

Ruff (0.14.10)

Loop control variable `pid` not used within loop body

Rename unused `pid` to `_pid`

(B007)